pkg/api/v1beta1/types.go

```diff
@@ -666,8 +684,6 @@ type ResourceList map[ResourceName]util.IntOrString
 // The name of the minion according to etcd is in ID.
 type Minion struct {
 	TypeMeta `json:",inline"`
-	// Queried from cloud provider, if available.
-	HostIP string `json:"hostIP,omitempty" description:"IP address of the node"`
```

Comment:
Thanks for the tip.  LegacyIP is for the purpose, we pick NodeAddresses[api.NodeLegacyHostIP] when we do the conversion.